docs/quickstart.md

```diff
@@ -1,70 +1,163 @@
-# Quick Start
+# Postgres
 
-This is brief guide to get you up and running with Red Sky Ops as quickly as possible.
+The purpose of this guide is to get you started with Red Sky Ops.
+In this example, you will use Red Sky Ops to tune the memory and cpu requirements for Postgres.
+We will cover:
+- Initializing Red Sky Ops in your cluster
+- Deploying the Postgres experiment to your cluster
+- Running trials using machine learning (ML) to determine the best configuration
 
-## Prerequisites
+If you encounter any issues or require assistance, please reach out to us via [slack][rso slack] or [contact us][contact us form].
 
-You must have a Kubernetes cluster. Additionally, you will need a local configured copy of `kubectl`. The Red Sky Ops Tool will use the same configuration as `kubectl` (usually `$HOME/.kube/config`) to connect to your cluster.
+## Prerequisites
 
-A local install of [Kustomize](https://github.com/kubernetes-sigs/kustomize/releases) (v3.1.0+) is required to build the objects for your cluster.
+- Kubernetes cluster
+  - A typical [minikube][minikube] cluster will be sufficient. The experiment may consume up to 4vcpu and 4GB memory
+- [kubectl][kubectl] properly configured for your cluster
+- [kustomize][kustomize] v3.1.0+
+- [redskyctl][install redskyctl]
+- [Red Sky Ops Account][rso account]
 
-If you are planning to create the postgres experiment from this guide, a [minikube](https://kubernetes.io/docs/setup/learning-environment/minikube/) cluster will be sufficient.
+## Initialize the Red Sky Ops Manager
 
-## Install the Red Sky Ops Tool
+After installing [redskyctl][install redskyctl] and creating an [account][rso account], log in to your Red Sky Ops account:
+```sh
+$ redskyctl login
+```
+You should see output similar to this:
```

Comment:
I get the need for positive feedback, but this is the same reason professional documentation writers do not like putting images into their documentation: it's too easy for the output to change thus ruining any established trust you previously gained by illustrating the output (i.e. "That's not exactly what I see, did I do it wrong?").

Maybe as a compromise we could consolidate the two blocks (without the "you should see") and fix preformatted blocks to use the `console` highlighter (instead of `sh`). The `console` highlighter is meant for mimicking shell sessions so it deals with the "prompt" the "command" and the "output".

Reply:
Doing testing locally, it doesn't seem like we have support for syntax highlighting enabled -- https://github.com/redskyops/redskyops-www/blob/staging/config.toml#L7-L8

Enabling it ( with the sh/console changes ) results in:

<img width="631" alt="image" src="https://user-images.githubusercontent.com/4570765/91330981-ca624980-e78f-11ea-8ee2-6a81b92ed370.png">

Which seems a little jarring after looking at the theme for so long, but I think I appreciate it better because it highlights the commands to run.

Reply:
I didn't try very hard with the [styles](https://github.com/redskyops/redskyops-www/blob/staging/themes/carbonrelay/static/css/docs.css).

Reply:
I think recent versions of Hugo changed the Markdown processor and syntax highlighter, so the default theme also changed; last I checked changing the config to have:
```
markup:
  highlight:
    style: bw
```
Produced more "boring" highlights.

Reply:
Dropping this for now as per https://github.com/redskyops/redskyops-controller/pull/143#discussion_r477528954